pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -450,7 +453,9 @@ public void start() throws PulsarServerException {
 
             this.defaultOffloader = createManagedLedgerOffloader(
                     OffloadPolicies.create(this.getConfiguration().getProperties()));
-
+            this.brokerInterceptor = BrokerInterceptors.load(config);
+            brokerService.setInterceptor(getBrokerInterceptor());
```

Comment:
We should support multiple broker interceptors.  Can you add a broker interceptor implementation that wraps a list of interceptors into one interceptor?

Reply:
The current implementation is supporting multiple broker interceptors, I added BrokerInterceptors.java also implement BrokerInterceptor interface.